<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAdminWin</class>
 <widget class="QDialog" name="MainAdminWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>851</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout2">
    <item>
     <widget class="QLabel" name="WelcomeLabel">
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000ff;&quot;&gt;Welcome &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout90">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WelcomeLabel_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Role: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ServerStateLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>530</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Server is connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QPushButton" name="PB_GetAccountNumber">
      <property name="text">
       <string>Get account number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewAccountBalance">
      <property name="text">
       <string>View accountBalance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewTransactionHistory">
      <property name="text">
       <string>View transaction history</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewBankDataBase">
      <property name="text">
       <string>View Bank DataBase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_CreateNewUser">
      <property name="text">
       <string>Create New User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_DeleteUser">
      <property name="text">
       <string>Delete User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_UpdateUser">
      <property name="text">
       <string>Update User</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>561</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="EmtyPage"/>
   <widget class="QWidget" name="GetAccountNumber">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Enter the user name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_GetAccountNumber"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="PB_GetAccounNumber">
          <property name="text">
           <string>Get Account Number</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="AccountNumberLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Enter the Account number: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_GetAccountBalance"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="PB_GetAccounBalance">
          <property name="text">
           <string>Get Account Balance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="BalanceLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="TV_ViewTransactionHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Enter the Account number: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LE_ViewTransactionHistory"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Count of transactions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_count"/>
        </item>
        <item>
         <widget class="QPushButton" name="PB_View">
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="TransactionHisLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="TV_ViewBankDataBase">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Enter the Account number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateUserID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Enter the full name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateFullName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Enter the UserName: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateUserName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_110">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Enter the account Password:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreatePassword"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Enter the balance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateBalance"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Enter the account Email: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateEmail"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Choose the Account type:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Checking</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Saving</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Enter the account signature: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CreateSignature"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Create">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CreateLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Enter the Account number: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_DeleteUserID"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_900">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Enter the Account signature: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_DeleteUserSignature"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_DeleteUserSignature">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DeleteUserLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>521</width>
       <height>206</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Enter the Account number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateUserID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Enter the account signature: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateSignature"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>What do you want to change: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="UpdateComboBox">
          <item>
           <property name="text">
            <string>AccountID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UserName</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EMail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Signature</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Enter the new value: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateNewValue"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="UpdateLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
